.lintstagedrc.mjs

```diff
@@ -1,3 +1,3 @@
 export default {
-  '!{yarn.lock,.husky/*,.*ignore}': ['prettier --write'],
+  '!{yarn.lock,.husky/*,.*ignore,*.png}': ['prettier --write'],
```

Comment:
Couldn't we somehow use the prettierignore? 

But there are no images defined either 🤷 

Else we can already ignore "all" image types.

Reply:
Yes, used `.prettierignore` instead. It is also applied to `--write`.